folium/plugins/time_slider_choropleth.py

```diff
@@ -26,21 +26,25 @@ class TimeSliderChoropleth(JSCSSMixin, Layer):
         Whether the Layer will be included in LayerControls.
     show: bool, default True
         Whether the layer will be shown on opening (only for overlays).
-
+    init_timestamp: int, default 0
```

Comment:
I find this name a bit confusing. It suggests it requires a timestamp, but instead it's an index. Could we rename it to something more self-explanatory? Like `initial_index` or `initial_timestamp_index`.